src/app/components/info-drawer/info-drawer.component.ts

```diff
@@ -74,11 +74,22 @@ export class InfoDrawerComponent implements OnChanges {
                 // workaround for Favorite files
                 this.loadNodeInfo(entry.id);
             } else {
-                this.displayNode = this.node.entry;
+                // workround Recent
+                if (!this.hasAspectNames(entry)) {
+                    this.loadNodeInfo(this.node.entry.id);
+                } else {
+                    this.displayNode = this.node.entry;
+                }
             }
         }
     }
 
+    private hasAspectNames(entry: MinimalNodeEntryEntity) {
+        return entry.content.mimeType.includes('image/')
+            && entry.aspectNames
+            && entry.aspectNames.includes('exif:exif');
```

Comment:
I think that we need aspect names only for images... so we should call loadNodeInfo if entry is image and it has **no** 'exif' aspect name